### PR TITLE
Fix payment methods not showing up

### LINKED ISSUE #19: Saved Cards under My-accounts in Stripe disappear on upgrading to WooCommerce 2.6.1 from WooCommerce 2.5.5
Prior to upgrade with **WooCommerce 2.5.5 with Stripe 3.0.2**

After performing the upgrade to **WooCommerce 3.6.1 with Stripe 3.0.2**

zd-woothemes-432332

**In the wild:**
https://wordpress.org/support/topic/saved-cards-versus-payment-methods-saved-cards-have-disappeared?replies=5#post-8539768

### PR DESCRIPTION
**Pull request by dwainm**
Fixes #19 .

#### Changes proposed in this Pull Request:
- Add a missing method to a class. This method is referenced in a filter but can't be reached so the filter returns a null value resulting in a blank payment methods page.

-------------------
- [x] Make sure your changes respect [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- [x] Did you make changes, or create a **new .js file**? If **Gruntfile.js** exists in the repo, make sure to run `grunt`.

### CODE REVIEW

**Comment by dwainm** [comment]
@roykho please review

**Comment by roykho** [comment]
LGTM!

**Comment by gedex** [comment]
This causes fatal error:

```
Fatal error: Cannot redeclare WC_Stripe::woocommerce_get_customer_payment_tokens() in /srv/www/wordpress-default/wp-content/plugins/woocommerce-gateway-stripe/woocommerce-gateway-stripe.php on line 461
```

File: woocommerce-gateway-stripe.php [hunk 1]
```diff
@@ -453,6 +453,38 @@ public static function log( $message ) {
 			error_log( $message );
 		}
 	}
+	/**
+	 * Gets saved tokens from API if they don't already exist in WooCommerce.
+	 * @param array $tokens
+	 * @return array
+	 */
+	public function woocommerce_get_customer_payment_tokens( $tokens, $customer_id, $gateway_id ) {
```
Comment (gedex) [inline woocommerce-gateway-stripe.php]: This method already declared above.

Comment (dwainm) [inline woocommerce-gateway-stripe.php]: @gedex interesting! on my localhost the function was missing. Inspecting locally again.

**Pull request merged**